public/docs/ts/latest/cookbook/i18n.jade

```diff
@@ -174,29 +251,66 @@ a#localization-folder
   <a href="https://msdn.microsoft.com/en-us/library/ee825488(v=cs.20).aspx" target="_blank">well-known codeset</a>.
 
   Move `messages.xlf` into the `locale` folder where it will become the source for all language-specific translations.
-  Then make a copy for the French language named `messages.fr.xlf` .
+  Then make a copy for the Spanish language named `messages.es.xlf` .
 
   Follow the same convention for each target language.
 
-  ### Translate
-  In the real world, you send the `messages.fr.xlf` file to a French translator who would fill in the translations
+  ### Translate text nodes
+  In the real world, you send the `messages.es.xlf` file to a Spanish translator who would fill in the translations
   using one of the 
   <a href="https://en.wikipedia.org/wiki/XLIFF#Editors" target="_blank">many XLIFF file editors</a>.
 
-  This sample file is easy to translate without a special editor or knowledge of French.
-  Open `messages.fr.xlf` and find the `<trans-unit>` section:
+  This sample file is easy to translate without a special editor or knowledge of Spanish.
+  Open `messages.es.xlf` and find the first `<trans-unit>` section:
 
-+makeExample('cb-i18n/ts/locale/trans-unit.html', '', 'locale/messages.fr.xlf (<trans-unit>)')(format=".")
++makeExample('cb-i18n/ts/locale/trans-unit.html', 'translate-hello', 'locale/messages.es.xlf (<trans-unit>)')(format=".")
 :marked
   This XML element represents the translation of the `<h1>` greeting tag you marked with the `i18n` attribute. 
   
   Using the _source_, _description_, and _meaning_ elements to guide your translation,
-  replace the `<target/>` tag with the French greeting:
-+makeExample('cb-i18n/ts/locale/messages.fr.xlf.html', 'trans-unit', 'locale/messages.fr.xlf (<trans-unit>, after translation)')(format=".")
+  replace the `<target/>` tag with the Spanish greeting:
++makeExample('cb-i18n/ts/locale/messages.es.xlf.html', 'translated-hello', 'locale/messages.es.xlf (<trans-unit>, after translation)')(format=".")
 :marked
   Note that the `id` is generated by the tool. Don't touch it.
   Its value depends on the content of the message and its assigned meaning.
   Change either factor and the `id` changes as well.
+
+  Translating the text nodes within the `<ng-container>`, `<!--i18n-->` and also the attributes with `i18n-x` are done in the same way:
+
++makeExample('cb-i18n/ts/locale/messages.es.xlf.html', 'translated-other-nodes', 'locale/messages.es.xlf (<trans-unit>)')(format=".")
+
+:marked
+  ### Translate plural and select
+  To translate a `plural` or a `select`, you provide the same ICU string but changing the matches:
```

Comment:
Sorry, is this the right comment for that sentence? I am not saying "same but different" in here.

I could say "To translate an ICU, provide...." but I prefer to say "plural" or a "select". same thing, yes, but I think people will understand it better if I use those words.